src/inmanta/app.py

```diff
@@ -585,6 +585,8 @@ def export(options: argparse.Namespace) -> None:
 
     if options.ssl:
         Config.set("compiler_rest_transport", "ssl", "true")
+    else:
+        Config.set("compiler_rest_transport", "ssl", "false")
```

Comment:
this is not OK, it will make it impossible to start the server with ssl 

if has to be a three way option: yes, no, unset.

Reply:
Oh. How did I miss that? Thanks!